rotkehlchen/chain/manager.py

```diff
@@ -430,14 +430,12 @@ def set_dot_rpc_endpoint(self, endpoint: str) -> Tuple[bool, str]:
     def activate_premium_status(self, premium: Premium) -> None:
         self.premium = premium
         for _, module in self.iterate_modules():
-            if getattr(module, 'premium', None):
```

Comment:
This change is still not okay. What you are doing right now will end up assigning a `premium` attribute to all iterated modules even if they do not have one.

What you want is to only assign it to modules that keep the premium attribute in memory and ignore all others.